Exercises/operators-and-type-conversions/operators-and-type-conversions.js

```diff
@@ -0,0 +1,21 @@
+// Operators/Type Conversation
+// 1. Get `100` using `+` operator and numeric value `10`.
+let stringWithNumber = 10 + '0';
+console.log(stringWithNumber);
+
+// 2. You have following string: `Amazon annual revenue of US$232.887 billion` extract revenue and count monthly income.
+let phrase = 'Amazon annual revenue of US$232.887 billion';
+
+let regularExpession = phrase.match(/([\d.]+)\s+(\S+)/);
```

Comment:
also you can rely on `$` symbol in this specific case and avoid regex, but it is good you know them :)

Reply:
@lizarusi, better will try with $ because I barely know regex, it was a high-level research that worked :(